Content.Shared/ItemRecall/SharedItemRecallSystem.cs

```diff
@@ -85,6 +85,8 @@ private void RecallItem(Entity<RecallMarkerComponent?> ent)
 
         _popups.PopupPredicted(Loc.GetString("item-recall-item-summon", ("item", ent)), actionOwner.Value, actionOwner.Value);
```

Comment:
```suggestion
        _popups.PopupClient(Loc.GetString("item-recall-item-summon", ("item", ent)), actionOwner.Value, actionOwner.Value);
```
While we are already at it we can also fix this one. Currently everyone sees the "the item appears in your hand" popup. With `popupClient` only the action user does.

Reply:
Agreed, I think there should be a third popup to others with "the item appears in X's hand". Lmk

Reply:
Good idea!
In that case you can switch to the other overload of `PopupPredicted` that takes two string arguments.